lumen/ai/agents.py

```diff
@@ -157,14 +157,31 @@ def _get_schema(self, source: Source | Pipeline, table: str = None):
         schema = dict(schema)
         for field, spec in schema.items():
             if "enum" in spec:
-                schema[field] = dict(spec, enum=spec["enum"][:5])
+                spec.pop("enum")
```

Comment:
How much of an effect does this have?

Reply:
I decided to pop it because I'm worried that it'll give the LLM context of only 5 values, which it might think are the only ones available.

Reply:
The formatter that is applied should be inserting an ellipsis to indicate that there are potentially more items.